testsuite/features/srv_delete_channel.feature

```diff
@@ -0,0 +1,82 @@
+# COPYRIGHT (c) 2019 SUSE LLC
+# License under the terms of the MIT License.
```

Comment:
"Licensed", no ?

I picked up a file at random and it reads:

`# Licensed under the terms of the MIT license.`

Reply:
I just changed. Thanks